packages/taiko-client/prover/proof_submitter/proof_submitter.go

```diff
@@ -170,14 +170,14 @@ func (s *ProofSubmitter) SubmitProof(
 		if err != nil {
 			return err
 		}
-		// TODO:  Need to check if there will be performance issues while using simple time.Sleep()
-		time.Sleep(submissionDelay)
+		delayTimer := time.After(submissionDelay)
```

Comment:
After a second thought, seems `time.AfterFunc()` is better?

Reply:
`time.AfterFunc()` needs to pass a function as a parameter, but I just want to continue with the rest of the code.